src/Symfony/Bridge/Twig/Extension/EmojiExtension.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Symfony\Component\Emoji\EmojiTransliterator;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+final class EmojiExtension extends AbstractExtension
+{
+    private static array $transliterators = [];
+
+    public function __construct(
+        private readonly string $defaultCatalog = 'text',
+    ) {
+        if (!class_exists(EmojiTransliterator::class)) {
+            throw new \LogicException('You cannot use the "emojify" filter as the "Emoji" component is not installed. Try running "composer require symfony/emoji".');
+        }
+    }
+
+    public function getFilters(): array
+    {
+        return [
+            new TwigFilter('emojify', $this->emojify(...)),
+        ];
+    }
+
+    /**
+     * Converts emoji short code (:wave:) to real emoji (👋)
```

Comment:
Do we need to pass the short code with the colons? As we are not parsing a text where the code is embedded, I suppose we could also support just `wave` instead of `:wave:`?

Reply:
We do yes: that's the convention used by all platforms. It prevents any ambiguities.

Reply:
> As we are not parsing a text where the code is embedded

Ah, actually yes we are parsing a text where short codes are embeded!

Reply:
I think this is a misunderstanding because the PR description only shows a string containing only the emoji